HCSStarRatingView/HCSStarRatingView.m

```diff
@@ -161,6 +167,38 @@ - (void)setFilledStarImage:(UIImage *)filledStarImage {
     }
 }
 
+
+- (void)setFilledStarColor:(UIColor *)filledStarColor {
+    if(_filledStarColor != filledStarColor) {
+        _filledStarColor = filledStarColor;
+        [self setNeedsDisplay];
+    }
+}
+
+- (void)setEmptyStarColor:(UIColor *)emptyStarColor {
+    if(_emptyStarColor != emptyStarColor) {
+        _emptyStarColor = emptyStarColor;
+        [self setNeedsDisplay];
+    }
+}
+
+- (void)setStarBorderColor:(UIColor *)startBorderColor {
+    if(_starBorderColor != startBorderColor) {
+        _starBorderColor = startBorderColor;
+        [self setNeedsDisplay];
+    }
+}
+
+- (void)setStarBorderWidth:(CGFloat)startBorderWidth {
+    if(startBorderWidth < 0){
+        startBorderWidth = 0;
+    }
+    
+    _starBorderWidth = startBorderWidth;
```

Comment:
Instead of the `if` above, why not just `_starBorderWidth = MAX(0, starBorderWidth)`?

What the `if` should be doing is validating if `_starBorderWidth != starBorderWidth`, similarly to the implementation in the other setters.